app/controllers/concerns/decidim/reporting_proposals/proposals_controller_override.rb

```diff
@@ -49,8 +49,19 @@ def complete
           redirect_to "#{Decidim::ResourceLocatorPresenter.new(@proposal).path}/preview" if reporting_proposal?
         end
 
+        def edit_form
+          form_attachment_model = form(AttachmentForm).from_model(@proposal.attachments.first)
+          @form = form_reporting_proposals_model
+          @form.attachment = form_attachment_model
+          @form
+        end
+
         private
 
+        def form_reporting_proposals_model
```

Comment:
```suggestion
        def form_proposal_model
```
If we override the original method, we don't need to override the method `edit_form`